README.Rmd

```diff
@@ -129,7 +129,7 @@ Install the package via `devtools` or CRAN:
 
 ## API Key
 
-To use the NASS Quick Stats API you need an [API key](https://quickstats.nass.usda.gov/api). The API key should in general not be included in scripts. One way of making the key available without defining it in a script is by setting it in your `.Renviron` file, which is usually located in your home directory. If you are an `rstudio` user, you can use `usethis::edit_r_environ()` to open your `.Renviron` file and add a line that looks like:
+To use the NASS Quick Stats API you need an [API key](https://quickstats.nass.usda.gov/api/). The API key should in general not be included in scripts. One way of making the key available without defining it in a script is by setting it in your `.Renviron` file, which is usually located in your home directory. If you are an `rstudio` user, you can use `usethis::edit_r_environ()` to open your `.Renviron` file and add a line that looks like:
```

Comment:
Did you use https://urlchecker.r-lib.org/ by the way? It's super handy.

Reply:
I did! This was the first time using it, very helpful.